lib/java/jvm-channel/src/main/java/org/enso/jvm/channel/Channel.java

```diff
@@ -26,8 +25,18 @@
 import org.graalvm.word.PointerBase;
 import org.graalvm.word.WordFactory;
 
-/** Channel connects two {@link JVM} instances. */
-public final class Channel implements AutoCloseable {
+/**
+ * Channel connects two {@link JVM} instances. A "channel" creates two identical instances of the
+ * {@code Channel} on both sides of the "channel" - e.g. in each of the JVMs. The instances are
+ * initialized with the same {@link Persistance.Pool}, so they both understand the same messages.
+ *
+ * @param <Data> internal data of the channel
+ */
+public final class Channel<Data extends Channel.Config> implements AutoCloseable {
+  private static final long ISOLATE_SVM = -1;
+  private static final long ISOLATE_MOCK_MASTER = -2;
+  private static final long ISOLATE_MOCK_SLAVE = -3;
```

Comment:
minor: The global consensus on naming changed few years ago. It is prefered to use master and worker, for obvious reasons.